config/locales/en.yml

```diff
@@ -107,17 +107,17 @@ en:
         one: stop
         other: stops
         type:
-          store: store
+          store: facility
           visit: visit
           rest: rest
-      store: Store
+      store: Facility
       stores:
-        one: Store
-        other: Stores
-      tag: Tag
+        one: Facility
+        other: Facilities
+      tag: Category
       tags:
-        one: tag
-        other: tags
+        one: Category
```

Comment:
Some words were starting with a small case. Did you've identified where it was involved ?

Reply:
Fixed, and also similar issues in other languages.